TYPO3.Neos/Classes/TYPO3/Neos/Command/SiteCommandController.php

```diff
@@ -63,6 +67,88 @@ class SiteCommandController extends CommandController
     protected $systemLogger;
 
     /**
+     * @Flow\Inject
+     * @var ContextFactoryInterface
+     */
+    protected $nodeContextFactory;
+
+    /**
+     * @Flow\Inject
+     * @var NodeTypeManager
+     */
+    protected $nodeTypeManager;
+
+    /**
+     * @Flow\Inject
+     * @var NodeService
+     */
+    protected $nodeService;
+
+    /**
+     * Create a new (blank) site
+     *
+     * This command allows to create a blank site with just a single empty document. The name of the site and the
+     * packageKey to assign to it must be specified.
+     *
+     * If no "--nodeName" is specified the command will create a unique node-name from the name of the site. If a node name is given
+     * it has to be unique.
+     *
+     * The specified "--nodeType" must it exists and have the superType "TYPO3.Neos:Document".
+     *
+     * If the flag "--active" is set the created site will be activated immediately.
+     *
+     * @param string $name The name of the site
+     * @param string $packageKey The site package
+     * @param string $nodeType The nodetype to use for the site node.
+     * @param string $nodeName The name of the site node. If no nodeName is given it will be determined from the siteName.
+     * @param boolean $active Activate the new site immediately.
+     * @return void
+     */
+    public function createCommand($name, $packageKey, $nodeType, $nodeName = null, $active = false)
+    {
+        if ($nodeName === null) {
+            $nodeName = $this->nodeService->generateUniqueNodeName('/sites', $name);
+        }
+
+        if ($this->siteRepository->findOneByNodeName($nodeName)) {
+            $this->outputLine('Error: A site with siteNodeName "%s" already exists', [$nodeName]);
```

Comment:
instead of `Error:` use `<error>A site ...</error>` to output red colored text..

same with the next two..